### ISSUE TITLE
Does it works with Resident evil Village preydog mod??

### ISSUE THREAD

**Issue by Masano91** [fholger/vrperfkit#110]

**Comment by fholger** [comment]
To my knowledge, Village is a DirectX 12 title, so no.

**Comment by Crimson-foxGITHUB** [comment]
It's also the only one of the Praydog mods to use DX12.

**Comment by SC7639** [comment]
That's sad. Is there any way to update this to use dx12 or be compatible with it. Having to download the dx11 re7 version to get a decent frame rate when in vr

**Comment by Crimson-foxGITHUB** [comment]
No. Would require the mod to be completely rewritten from scratch.
